src/main/kotlin/org/rust/lang/core/psi/ext/RsQualifiedNamedElement.kt

```diff
@@ -40,11 +40,80 @@ class RsQualifiedName private constructor(
         return segments.joinToString(separator = "/", postfix = anchor)
     }
 
+    override fun equals(other: Any?): Boolean {
```

Comment:
I suspect that we can't just use data class here? It would be nice to leave a comment, why so.

Reply:
fixed